astroquery/ned/core.py

```diff
@@ -704,6 +704,23 @@ def _parse_result(self, response, verbose=False):
 Ned = NedClass()
 
 
+def _parse_radius(radius):
```

Comment:
There are test units that are using this. I was hoping this had some functionalities for these two particular API's so I thought better keep them back. Working on it.